documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -4,17 +4,16 @@
 // getting-started.adoc
 
 [id='setup-external-clients-{context}']
-= Setting up access for clients outside of Kubernetes
+= Setting up client access to a Kafka cluster
 
 [role="_abstract"]
+Using the address of the Kafka cluster, you can provide access to a client in the same cluster; or provide external access to a client on a different Kubernetes namespace or outside Kubernetes entirely.
 This procedure shows how to configure client access to a Kafka cluster from outside Kubernetes.
```

Comment:
This talks only abotu outside Kubernetes. I think that maybe we can clarify it as _outside of Kubernetes_ or _from other Kubernetes cluster_?